test/SAMtags/README.md

```diff
@@ -0,0 +1,35 @@
+MM and ML auxiliary tags
+========================
+
+The purpose of these test files is to test parsing of the MM and ML
+tags.  These succint MM and ML tags are present in the .sam files,
+with a more human readable expanded form in the .txt files.
```

Comment:
.txt and .dat files

Reply:
Ugh I forgot to git rm the .dat files when I produced the new .sam versions.  That was my intention, and I thought yours too.  I'll fix it.